apps/explorer/components/ExplorerDatum.tsx

```diff
@@ -48,13 +50,16 @@ export function ExplorerDatum({
             <ValueCopyable
               scaleSize="18"
               label={entityType}
```

Comment:
Looks like an existing bug getting surfaced, but I noticed this label being set to `label={entityType}` is causing the custom label to override `getEntityTypeCopyLabel`, I think we can safely remove this line and fix that issue.

<img width="516" alt="Screenshot 2024-06-18 at 10 48 22 AM" src="https://github.com/SiaFoundation/web/assets/1412796/e486b065-3abe-4946-a350-04d863c785a5">

Reply:
Good catch! Got it.